zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
```

Comment:
I originally had the constants at the beginning of the functions, but then I had the same set of constants at the beginning of each. I promoted them to the global scope when I realized they were common to the whole file.